docs/management/gcache-write-set-cache-encryption.md

```diff
@@ -0,0 +1,112 @@
+# GCache encryption and Write-Set cache encryption
+
+These features are [tech preview](../glossary.md#tech-preview). Before using these features in production, we recommend that you test restoring production from physical backups in your environment, and also use the alternative backup method for redundancy.
+
+## GCache and Write-Set
+
+In Percona XtraDB Cluster, 
+
+## GCache and Write-Set cache encryption
+
+Enabling this feature encrypts the Galera GCache and Write-Set cache files with a File Key.  
+ 
+  GCache has a RingBuffer on-disk file to manage write-sets. The keyring only stores the Master Key and the File Key used by the Ring Buffer file. The Ring Buffer file of GCache is non-volatile, which means this file survives a restart. The File Key is not stored for GCache off-pages and Write-Set cache files.
```

Comment:
The keyring stores only MK.
The keyring only stores the Master Key which is used to encrypt the File Key used by the Ring Buffer file. The encrypted File Key is stored in the Ring Buffer's preamble.

Reply:
Rewritten.